DependencyInjection/Compiler/DescriptionEnhancerPass.php

```diff
@@ -26,16 +26,26 @@ public function process(ContainerBuilder $container)
 
         $taggedIds = $container->findTaggedServiceIds('cmf_resource.description.enhancer');
         $enabledEnhancers = $container->getParameter('cmf_resource.description.enabled_enhancers');
+        $validAttributes = ['name', 'alias', 'priority'];
```

Comment:
Is it really worth it? Seems overkill to me - the exception messages would be less explicit and the there is not much code here.